eda-frontend/src/redux/actions/authActions.js

```diff
@@ -154,8 +156,8 @@ export const login = (username, password, toUrl) => {
             window.open('', '_self')
             window.close()
           } else {
-            window.open(toUrl, '_self')
             localStorage.setItem('ard_redurl', '')
+            window.location.href = toUrl+"?token=" + localStorage.getItem('esim_token');
```

Comment:
Maybe a security concern? 
Normally I wouldn't want to expose the token like this via the URL